src/rebar_covertool.erl

```diff
@@ -1,8 +1,5 @@
 -module(rebar_covertool).
 
-%% rebar3 plugin callback
--export([init/1]).
```

Comment:
If you leave this here, it seems like it would preserve backwards compatibility, which seems like a good idea. Maybe log a deprecation warning (`rebar_api:warn/2`) like: 

> The 'covertool' plugin is being referenced by deprecated name 'rebar_covertool'. Use 'covertool' instead in your rebar.config file. See the covertool README for more information.